debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/connection/ConnectionMode.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb.connection;
+
+import io.debezium.config.EnumeratedValue;
+
+/**
+ * The set of predefined MongoDbConnectionMode options or aliases.
+ */
+public enum ConnectionMode implements EnumeratedValue {
```

Comment:
Why is not this enum inner in MongoDbConnectorConfig as the others? Is there anything special about it?